source/client-side-encryption/client-side-encryption.rst

```diff
@@ -550,10 +553,43 @@ the following process:
       AWSKMSOptions_ map from :math:`C` and insert that map onto :math:`P` as
       the ``aws`` property.
 
-3. Return :math:`P` as the additional KMS providers to libmongocrypt_.
+3. If the user-provided kmsProviders_ (from ClientEncryptionOpts_ or
+   AutoEncryptionOpts_) contains an ``gcp`` property, and that property is an
+   empty map:
+
+   1. Attempt to obtain credentials :math:`C` from the environment
```

Comment:
Does the schema of `GCPKMSOptions` need to be updated to note the `accessKey` property? From this doc alone, it's not clear how to create the GCP KMS object using the arbitrary string of the access token.

Reply:
Great catch. `GCPKMSOptions` has been updated to be either `GCPKMSCredentials` or the new `GCPKMSAccessToken` interface.